provisioning/ubuntu1804_preseed.cfg

```diff
@@ -0,0 +1,61 @@
+# Preseeding only locale sets language, country and locale.
+d-i debian-installer/locale string en_US
+
+# Keyboard selection.
+d-i console-setup/ask_detect boolean false
+d-i keyboard-configuration/xkb-keymap select de-CH
```

Comment:
Please switch to the `en` / `us` keyboard layout thus we are aligned with the other existing images